generators/client/templates/angular/package.json.ejs

```diff
@@ -47,7 +47,7 @@
     "core-js": "2.6.5",
     "moment": "2.24.0",
     "ng-jhipster": "0.9.3",
-    "ngx-cookie": "4.1.2",
```

Comment:
Can you try to add @nguniversal/express-engine to peer dependencies?
see: https://github.com/salemdar/ngx-cookie/commit/1956eff8940859147268f5b2b6cc3e7619a89247

Reply:
It was tried before in https://github.com/jhipster/generator-jhipster/pull/9354 and reverted as this peer dependency is more appropriate for server-side rendering. There is an associated open issue on that project that was also linked, but, don't see much progress on that. I am of the opinion to move away from this dependency, but, maybe in separate PR.